### PR TITLE
ruby script

### PR DESCRIPTION
**Pull request by Chekushkin**

### CODE REVIEW

**Comment by alisnic** [comment]
:+1:

File: ruby/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Example of integrating Salt Edge API with Ruby.
+
+```
+api = Saltedge.new("CLIENT_ID", "SERVICE_SECRET", "private_pem_path")
+api.request("GET", "https://www.saltedge.com/api/v2/countries")
+api.request("POST", "https://www.saltedge.com/api/v2/customers/", {\"data\":{\"identifier\":\"my_unique_identifier\"}})
```
Comment (idanci) [inline ruby/README.md]: `api.request("POST", "https://www.saltedge.com/api/v2/customers/", {:data => { :identifier => "my_unique_identifier"}}.to_json)`

File: ruby/saltedge.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    @client_id        = client_id
+    @service_secret   = service_secret
+    @private_pem_path = private_pem_path
+  end
+
+  def request(method, url, params={})
+    hash = {
+      method:     method,
+      url:        url,
+      expires_at: (Time.now + 60).to_i,
```
Comment (deltadd) [inline ruby/saltedge.rb]: 60 constant

Comment (deltadd) [inline ruby/saltedge.rb]: .minutes

File: ruby/saltedge.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      params:     !params.empty? ? params.to_json : ""
```
Comment (deltadd) [inline ruby/saltedge.rb]: extract method

File: ruby/saltedge.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    RestClient::Request.execute(
+      method:  method,
+      url:     url,
+      payload: hash[:params],
+      headers: {
+        "Accept"         => "application/json",
+        "Content-type"   => "application/json",
+        "Expires-at"     => hash[:expires_at],
+        "Signature"      => signature(hash).delete("\n"),
```
Comment (deltadd) [inline ruby/saltedge.rb]: move .delete("\n") to signature method

File: ruby/saltedge.rb (hunk 2 above)
Comment (idanci) [inline ruby/saltedge.rb]: Requiring active_support only for that does not seem to be a good idea

Comment (deltadd) [inline ruby/saltedge.rb]: agree

**Pull request merged**